spec/support/common_basic_reviewable_serializer.rb

```diff
@@ -38,4 +38,12 @@
       expect(subject[:flagger_username]).to eq("gg.osama")
     end
   end
+
+  describe "#created_at" do
+    it "equals the reviewable's created_at timestamp" do
```

Comment:
I don't think we're testing that it is equal but rather the serialized object contains a field with the object's `created_at` property.

Reply:
Are you saying we don't need to test for equality and just testing for the presence of `created_at` is enough? I don't disagree with that, but right now there's an assertion for equality so I'm a little bit confused.

Reply:
ok I've updated the desc!